application/jaxrs-hazelcast-quarkus/src/test/java/webapp/tier/service/socket/HazelcastSocketTestForQuarkus.java

```diff
@@ -0,0 +1,64 @@
+package webapp.tier.service.socket;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.net.URI;
+import java.util.concurrent.LinkedBlockingDeque;
+import java.util.concurrent.TimeUnit;
+
+import javax.websocket.ClientEndpoint;
+import javax.websocket.ContainerProvider;
+import javax.websocket.OnMessage;
+import javax.websocket.OnOpen;
+import javax.websocket.SendHandler;
+import javax.websocket.SendResult;
+import javax.websocket.Session;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.common.http.TestHTTPResource;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+class HazelcastSocketTestForQuarkus {
+
+	private static final LinkedBlockingDeque<String> MESSAGES = new LinkedBlockingDeque<>();
+
+	@TestHTTPResource("/quarkus/hazelcast/subscribe")
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6010272)